source/_integrations/tasmota.markdown

```diff
@@ -2,6 +2,7 @@
 title: Tasmota (beta)
 description: Instructions on how to integrate Tasmota with Home Assistant.
 ha_category:
+  - Automation
```

Comment:
I doubt whether automation is a correct category label for this integration 🤔

Reply:
It generates automation triggers.

Reply:
Hmm, what are your ideas about this @frenck. When does something fall under the automations category?

Reply:
It  took a bit of thinking before adding the automation tag since our integration doesn't fall in automation as strict term but just generates triggers for them. 
Your call if needed or not.

Reply:
After a chat with Erik we decided to remove the `automation` category.

Reply:
Maybe we should add a new tag for integrations which provide custom device automations, i.e. not just what is provided from base components?

Reply:
If we add a marker for an integration providing certain features, we should consider adding a frontmatter option (not a category). The removal of this category is right.